packages/trpc/server/routers/viewer/organizations/utils.ts

```diff
@@ -0,0 +1,83 @@
+import { isOrganisationAdmin } from "@calcom/lib/server/queries/organisations";
+import { prisma } from "@calcom/prisma";
+import type { Prisma } from "@calcom/prisma/client";
+import { MembershipRole } from "@calcom/prisma/enums";
+
+import { TRPCError } from "@trpc/server";
+
+import type { TrpcSessionUser } from "../../../trpc";
+import type { TAddBulkTeams } from "./addBulkTeams.schema";
+
+interface AddBulkToTeamProps {
+  user: NonNullable<TrpcSessionUser>;
+  input: TAddBulkTeams;
+}
+
+export const addBulkToTeam = async ({ user, input }: AddBulkToTeamProps) => {
```

Comment:
Could this function have a better name? It's called `addBulkToTeam` but it's not really clear what `Bulk` is.